package.json

```diff
@@ -31,10 +31,14 @@
     "lint:ts": "tsc --noEmit",
     "prebuild": "rimraf dist",
     "build:copy-files": "babel-node ./scripts/copy-build-files.js",
-    "prepublishOnly": "yarn-or-npm build",
+    "prepublishOnly": "pinst --disable",
```

Comment:
we absolutely need to run `build` before publishing to ensure we publish the latest version.

Reply:
Oups that was absolutely not intended !